kong-3.6.0-0.rockspec

```diff
@@ -198,6 +198,7 @@ build = {
     ["kong.workspaces"] = "kong/workspaces/init.lua",
 
     ["kong.db"] = "kong/db/init.lua",
+    ["kong.db.sort"] = "kong/db/sort.lua",
```

Comment:
Should we rename it to `kong/db/utils.lua`? It sounds more reasonable.

Reply:
Reasonable. Renamed.